sdk/astronomer/Azure.ResourceManager.Astro/CHANGELOG.md

```diff
@@ -0,0 +1,17 @@
+# Release History
+
+## 1.0.0-beta.1 (Unreleased)
```

Comment:
Specify the release date here by replacing `Unreleased`, the format is `yyyy-mm-dd`.

Reply:
@ArthurMa1978 Is it the date when service went to GA or the date when we want to release dot not sdk?

Reply:
Latter

Reply:
done